project-clone/main.go

```diff
@@ -38,7 +38,6 @@ const (
 	tmpLogFilePath = "/tmp/" + logFileName
 )
 
-// TODO: Handle sparse checkout
 func main() {
```

Comment:
You might consider also moving `GetClonePath` to the projects library, since we're doing it both in `container` and project-clone

Reply:
This comment is here because it's the only file in `project-clone` in this PR, this has nothing to do with removing the TODO (thanks!) :D

Reply:
I ended up moving `GetClonePath` since it made sense anyhow :P